library/Fission/CLI/Auth.hs

```diff
@@ -14,11 +15,20 @@ import qualified System.Console.ANSI as ANSI
 
 import qualified Data.Yaml as YAML
 import           Servant
+import           Servant.Client
 
 import           Fission.Internal.Constraint
 import           Fission.Internal.Orphanage.BasicAuthData ()
 import qualified Fission.Internal.UTF8 as UTF8
 
+newtype CLIError = CLIError { message :: Text }
```

Comment:
For consistency, let's:

1. Move this to it's own `*.Error` module
2. More structured errors, if that's useful

I'm going to just... do this? At worst, we can roll it back